packages/account/src/Components/formik-confirmation-checkbox/__tests__/formik-confirmation-checkbox.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { FormikConfirmationCheckbox } from '../formik-confirmation-checkbox';
+import { Formik, Form } from 'formik'; // Import Formik and Form
+
+describe('ConfirmationCheckbox', () => {
+    test('renders checkbox with label', () => {
+        const props = {
+            confirmed: true,
+            setConfirmed: jest.fn(),
+            label: 'I confirm my details are correct.',
+        };
+
+        render(
+            <Formik initialValues={{}} onSubmit={jest.fn()}>
+                <Form>
+                    <FormikConfirmationCheckbox {...props} />
+                </Form>
+            </Formik>
+        );
+
+        const checkbox = screen.getByLabelText('I confirm my details are correct.');
+        expect(checkbox).toBeInTheDocument();
+    });
+
+    test('calls setConfirmed function when checkbox is clicked', () => {
+        const setConfirmedMock = jest.fn();
+        const props = {
+            confirmed: true,
+            setConfirmed: setConfirmedMock,
+            label: 'I confirm my details are correct.',
+        };
```

Comment:
why don't we make it into one standard props, then in fireEvent click, just check for `expect(props.setConfirmed).toHaveBeenCalled();`

Reply:
Sure thing. Updated 👍